tests/congruence_tests/test_query.py

```diff
@@ -1294,3 +1308,18 @@ def test_query_group(prefer_grpc):
         except AssertionError as e:
             print(f"\nFailed with filter {query_filter}")
             raise e
+
+
+@pytest.mark.parametrize("prefer_grpc", [False, True])
+def test_random_sampling(prefer_grpc):
+    fixture_points = generate_fixtures(100)
+
+    searcher = TestSimpleSearcher()
+
+    local_client = init_local()
+    init_client(local_client, fixture_points)
+
+    remote_client = init_remote(prefer_grpc=prefer_grpc)
+    init_client(remote_client, fixture_points)
+
+    compare_client_results(local_client, remote_client, searcher.random_query)
```

Comment:
Why are the random implementations between local and server equivalent?
I would expect those to return different values.

Reply:
In this test we basically sample all the points available in the collection
In `random_query` we use `limit=100` and we generate 100 points in `fixture_points = generate_fixtures(100)`

Reply:
We return all points postprocessed to be sorted by ID. Just to make sure we can return all of them